lib/challenge.js

```diff
@@ -1,305 +1,338 @@
+// prompt8();
+
+// 🌟 MAKE A COMMIT: "Complete prompt 8"
+
+/*
+ * Prompt 9:
+ *
+ * Iterate over the starWars array printing each character's name and index
+ *
+ * i.e.:
+ *  0. Han
+ *  1. C3PO
+ *  2. R2D2
+ */
+
+const starWars = ["Han", "C3PO", "R2D2", "Luke", "Leia", "Anakin"];
+
+function prompt9() {
+  for (let i = 0; i < starWars.length; i++) {
+    console.log(starWars[i]);
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt9();
+
+// 🌟 MAKE A COMMIT: "Complete prompt 9"
+
+/*
+ * Prompt 10:
+ *
+ * Write a loop that pushes every even number between 0 and 100 into an array,
+ * then print the array.
+ */
+function prompt10() {
+  const myArray = [];
+  for (let i = 0; i <= 100; i ++) {
+    console.log(i);
+  return myArray;
+}
+}
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// console.log(prompt10());
+
+// 🌟 MAKE A COMMIT: "Complete prompt 10"
+
+/*
+ * Prompt 11:
+ *
+ * Find the median number in the following nums array (the mid point), then
+ * console.log that number. The median number is the middle number in a SORTED list of numbers. Think through the steps you'll need to take to determine this number with code!
+ *
+ * Hint: What does Math.floor() do?
+ */
+
+let nums = [
+  14,
+  11,
+  16,
+  15,
+  13,
+  16,
+  15,
+  17,
+  19,
+  11,
+  12,
+  14,
+  19,
+  11,
+  15,
+  17,
+  11,
+  18,
+  12,
+  17,
+  12,
+  71,
+  18,
+  15,
+  12,
+];
+
+function prompt11() { 
+  let median; 
+  nums.sort((a, b) => {return a - b});
+  i = (nums.length) /2;
+  i = Math.floor(i)
+  median = nums[i]
+  console.log(median)
+   
+ }
+  
+  // prompt11();
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt11();
+
+// 🌟 MAKE A COMMIT: "Complete prompt 11"
+
+/*
+ * Prompt 12:
+ *
+ * Did you know that you can nest loops? The catch is that you have to use
+ * different incrementers (normally i) in the inner vs. outer loops. You'll commonly see developers use
+ * another 1-letter incrementer (j, k, l, etc).
+ *
+ * Create an "outer" loop that counts up from 1 to 10 with an incrementer of i.
+ * Create an "inner" loop that counts from 11 to 20 with an incrementer of j.
+ * Inside the inner loop, print `i: ${i} / j: ${j}`
+ * You might need to consult Google to see examples of nested for loops!
+ */
+
+function prompt12() {
+  for (let i = 1; i < 11; i++) {
+    for (let j = 11; j < i + 11; j++) {
+      console.log(`i: ${i} / j: ${j}`);
+    }
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt12();
+
+// 🌟 MAKE A COMMIT: "Complete prompt 12"
+
+/*
+ * Prompt 13:
+ *
+ * Using nested loops, print all the values inside the nested arrays.
+ */
+
+let nestedArrays = [
+  ["a", "b", "c"],
+  ["d", "e", "f"],
+  ["g", "h", "i"],
+];
+
+function prompt13() {
+  for(i=0;i<nestedArrays.length;i++){
+    for(j=0;j<nestedArrays[i].length;j++){
+      console.log(nestedArrays[i][j]);
+    }
+  }
+}

```

Comment:
Keep an eye on the spacing around the operators and brackets.  To make this more readable I would rewrite this as:
```
function prompt13() {
  for (i = 0; i < nestedArrays.length; i++) {
    for (j = 0; j < nestedArrays[i].length; j++) {
      console.log(nestedArrays[i][j]);
    }
  }
}
```